content/zh/docs/tasks/manage-daemon/update-daemon-set.md

```diff
@@ -234,6 +234,17 @@ daemonset "fluentd-elasticsearch" successfully rolled out
 
 ### DaemonSet 滚动更新卡住
 
+<!--
+Sometimes, a DaemonSet rolling update may be stuck. Here are some possible
+causes:
+
+#### Some nodes run out of resources
+-->
+有时，DaemonSet 滚动更新可能卡住，以下是一些可能的愿意：
+
+
+### 一些节点可用资源耗尽
```

Comment:
```suggestion
#### 一些节点可用资源耗尽
```

Reply:
Thank you for suggestion
suggestion applied